README.tpl

```diff
@@ -1,42 +0,0 @@
-**_Please be aware that version `v0.4.0` of rusty-cheddar will be based around the more general
```

Comment:
I'd rather keep this.  It's used with cargo-readme to generate the README.md and saves having to update documentation in both README.md *and* the code, since cargo-readme extracts it for you.